test/rating.js

```diff
@@ -16,40 +17,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat(window.GoogleMusic.Playback.SINGLE_REPEAT);
```

Comment:
More history can be found here:

https://github.com/radiant-player/radiant-player-mac/issues/252